### PR TITLE
HashUtil solution

### PR DESCRIPTION
**Pull request by Victoria-Moiseienko**

### CODE REVIEW

File: src/main/resources/init_db.sql [hunk 1]
```diff
@@ -142,3 +142,6 @@ ADD CONSTRAINT `user_id_fk`
   ALTER TABLE `internet_shop`.`orders`
 ADD COLUMN `deleted` TINYINT(1) NOT NULL DEFAULT 0 AFTER `user_id`;
 
+ALTER TABLE `internet_shop`.`users`
+ADD COLUMN `salt` VARBINARY(16) NOT NULL AFTER `deleted`;
```
Comment (AnaLanda) [inline src/main/resources/init_db.sql]: I think that in this file you can just add this column in your original `CREATE TABLE `internet_shop`.`users`` block :)

**Review by AnaLanda — COMMENTED** [review]
Looks great! 😃

File: src/main/resources/init_db.sql (hunk 1 above)
Comment (dmytro-hryhoruk) [reply src/main/resources/init_db.sql]: agree

Comment (dmytro-hryhoruk) [reply src/main/resources/init_db.sql]: let's modify the original statement

**Review by dmytro-hryhoruk — APPROVED** [review]

File: src/main/java/com/internet/shop/model/User.java [hunk 2]
```diff
@@ -15,11 +16,12 @@ public User(String name, String login, String password) {
...
     }
 
-    public User(Long id, String name, String login, String password, Set<Role> roles) {
+    public User(Long id, String name, String login, String password, byte[] salt, Set<Role> roles) {
         this.id = id;
         this.name = name;
         this.login = login;
         this.password = password;
+        this.salt = salt;
         this.roles = roles;
```
Comment (dmytro-hryhoruk) [inline src/main/java/com/internet/shop/model/User.java]: image
let's modify your constructors like this

Comment (Victoria-Moiseienko) [reply src/main/java/com/internet/shop/model/User.java]: Ok. Done

File: src/main/resources/init_db.sql (hunk 1 above)
Comment (Victoria-Moiseienko) [reply src/main/resources/init_db.sql]: Ok. Done

**Review by dmytro-hryhoruk — COMMENTED** [review]
I think it's time to configure travis in your project https://www.youtube.com/watch?v=BHSsWTbuow4&ab_channel=Mateacademy

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**